### DISCUSSION TITLE
Xdrip and libre 2

### DISCUSSION THREAD

**Discussion by Electrik-rich546456** [NightscoutFoundation/xDrip#2864] [General]
Hi, I wonder if someone can tell me why my graph is showing two  different values for a given time ? I'm using libre 2 with oop2 
See eg below.
![Screenshot_20230530-221153](https://github.com/NightscoutFoundation/xDrip/assets/58092123/11cc8cce-6023-421f-8536-e86686ed257c)

**Comment by Navid200** [comment]
That's raw data you have enabled.
So, you are seeing what is coming from the sensor, and you are also seeing what has been calibrated.

If you look at display settings:  
Settings &#8722;> xDrip+ Display Settings &#8722;> Graph Settings

There, you can see a settings like 
Display raw data plot or
Show Libre trend

Disable one or the other and see what each will do and please tell us.
